dissect/target/tools/info.py

```diff
@@ -50,8 +50,8 @@ def main():
     parser.add_argument("-d", "--delimiter", default=" ", action="store", metavar="','")
     parser.add_argument("-s", "--strings", action="store_true", help="print output as string")
     parser.add_argument("-r", "--record", action="store_true", help="print output as record")
-    parser.add_argument("-j", "--json-pretty", action="store_true", help="output records as pretty json")
-    parser.add_argument("-J", "--json", action="store_true", help="output records as one-line json")
+    parser.add_argument("-j", "--json", action="store_true", help="output records as pretty json")
+    parser.add_argument("-J", "--jsonlines", action="store_true", help="output records as one-line json")
```

Comment:
@Schamper This change breaks `target-info`. At lines 75 and 77 `args.json` needs to be replaced with `args.jsonlines` and `args.json_pretty` needs to be replaced with `args.json`.

Reply:
You are correct, too hastily merged :) Fixed with #186